viahtml/pywb_config.yaml

```diff
@@ -2,10 +2,10 @@ collections:
     "proxy":
         index: $live
         templates:
-            banner: templates/banner.rewriter
             error: templates/error.rewriter
             head_insert: templates/head_insert.rewriter
```

Comment:
I'm not clear on what the `.rewriter` suffix does here. The actual files on disk have `.html` suffixes and it seems that they get rendered just fine if referencing `templates/customer_banner.html` rather than `templates/custom_banner.rewriter`.

Reply:
I asked around on Slack, but we're not clear on where this came from originally. We might be able to revert to using a `.html` suffix to avoid confusion.

> I don't remember why the files had a weird extension. This is most likely a result of trial and error at the time. There weren't really any particularly detailed docs about it